securedrop_client/api_jobs/downloads.py

```diff
@@ -166,7 +168,7 @@ def call_download_api(self, api: API, db_object: Reply) -> Tuple[str, str]:
         sdk_object.source_uuid = db_object.source.uuid
         return api.download_reply(sdk_object)
 
-    def call_decrypt(self, filepath: str, session: Session = None) -> None:
+    def call_decrypt(self, filepath: str, session: Session = None) -> str:
```

Comment:
It would be helpful to update the comment for this method in the `DownloadJob` base class to explain what kind of string it should return since calling decrypt will return both the original filename and filepath strings. It would be helpful to know that we want to get the decrypted original filename so we can store it in the local db.

Reply:
Agreed. Whatever we decide to return from `decrypt_submission_or_reply` should be explained here.